packages/utilities/psammead-assets/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+<h1 align="center">Psammead Assets Changelog</h1>
```

Comment:
```suggestion
#Psammead Assets Changelog#
```
To align this changelog with the one in https://github.com/BBC-News/psammead/pull/63/files

Reply:
It doesnt need the second hash, will do manually